react/features/device-selection/DeviceSelectionPopup.js

```diff
@@ -164,10 +166,13 @@ export default class DeviceSelectionPopup {
      * Returns Promise that resolves with true if the device change is available
      * and with false if not.
      *
+     * @param {string} [deviceType] - Values - 'output', 'input' or undefined.
+     * Default - 'input'.
```

Comment:
I think even if you pass 'input' it will return the input devices. I think for the purpose of documenting the function this way is better and clearer because otherwise we have to explain that we sending a message, how this message is translated, etc.